### PR TITLE
Not allocating memory in default bank in hw_emu & data_flow fixes in hw_emu

### PR DESCRIPTION
**Pull request by chvamshi-xilinx**
Hi,

This change contains a fix to not allocate memory in default DDR bank in hw_emu.

Thanks
Vamshi Krishna

### CODE REVIEW

**Comment by stsoe** [comment]
ok to test

**Comment by chvamshi-xilinx** [comment]
retest this please

**Comment by maxzhen** [comment]
retest this please

File: src/runtime_src/driver/common_em/em_defines.h [hunk 1]
```diff
@@ -190,12 +190,13 @@ namespace xclemulation {
     int                   fd;
   };
 
-  static inline unsigned xocl_bo_ddr_idx(unsigned flags)
+  //we should not create a memory in default bank for hw_emu. As sw_emu doesnt have rtd information, we are not doing any error check
```
Comment (keryell) [inline src/runtime_src/driver/common_em/em_defines.h]: Line too long

**Pull request merged**